docs/learn/learn-nominator.md

```diff
@@ -49,6 +49,65 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Bags-list
+
+Nominating accounts are placed in a semi-sorted list called bags-list. This sorting functionality is extremely 
+important for the [long-term improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) 
+of the staking/election system. Bags-list allows up to 50,000 nominators to set their *intention* to nominate, 
+of which, the stake of the top 22,500 nominators is applied to the validators in the active set. The bags-list
+can be previewed on [Polkadot JS Apps > Network > Staking > Bags > All Bags](https://polkadot.js.org/apps/#/staking/bags).
+
+![Bags list](../assets/staking/bags-list.png)
+
+:::info Minimum DOT required to earn staking rewards
+
+Minimum DOT required to submit intent to nominate is 10 DOT, but the minimum active nomination required to earn
+staking rewards is dynamic and may be much higher, which can be viewed on [Polkadot JS Apps > Network > Staking > Targets page](https://polkadot.js.org/apps/#/staking/targets).
+
+:::
+
+![Minimum Active Nomination](../assets/staking/min-active-nomination.png)
+
+Bonding additional tokens or unbonding the staked tokens will automatically place the nominating account in the 
+appropriate bag. While the system tries its best to ensure nominators are always represented in the correct bag, 
+certain changes in bonded funds (e.g. a slash in the negative direction, or rewards in the positive direction) can 
+cause an account to be in the wrong bag, and for operational safety reasons the system will not automatically
+self-adjust.
+
+:::caution `bagsList.putInFrontOf` and `bagsList.rebag` extrinsics
+
+The nominator accounts in a bag are sorted based on their insertion order, not by their nomination stake. 
+`bagsList.putInFrontOf` extrinsic can be issued to move up in the bag, which might be very useful for the 
+accounts in the last bag eligible for receiving staking rewards. Also, balance changes due to staking rewards or 
```

Comment:
You talk about the last bag here, which is an important concept, but you have not explained it. I would include this explanation + some graphics about it beforehand as well. Example of what I have in mind is the "bags-list in depth" here: https://docs.google.com/document/d/1RMoTpvXyoudMeVTRJ-a6p9XDpdCfUovRapJINZJB4hI/edit#

Reply:
Noted

Reply:
Okay I see some explanation along those lines further below, but I think it is better to make it more comprehensive and move it to the top.